style.css

```diff
@@ -417,3 +348,507 @@ footer h4 {
+  }
+
+  .work1 {
+    grid-column: 2 / span 4;
+    grid-row: 1 / span 2;
+  }
+
+  .work2 {
+    grid-column: 1;
+    grid-row: 4;
+  }
+
+  .work3 {
+    grid-column: 1 / 3;
+    grid-row: 5 / span 2;
+  }
+
+  .work4 {
+    grid-column: 4 / 6;
+    grid-row: 6 / 7;
+  }
+
+  .work5 {
+    grid-column: 3 / span 2;
+    grid-row: 8;
+  }
+
+  .work6 {
+    display: none;
+  }
+
+  .info5 {
+    visibility: hidden;
+  }
+
+  .info11 {
+    height: 100%;
+    margin-top: -358px;
+    padding: 60px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .info4 {
+    height: 100%;
+    padding: 30px 24px 30px 30px;
+    display: flex;
+    justify-content: space-between;
+    margin-top: -350px;
+  }
+
+  .info2 {
+    display: none;
+  }
+
+  .info3 {
+    display: none;
+  }
+
+  /* info wor1 */
+  .project-name1 {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 60px;
+    width: 100%;
+    margin: 0 auto 34px 20px;
+    padding: 0;
+  }
+
+  .box11 {
+    display: flex;
+    gap: 20px;
+    margin-bottom: 60px;
+  }
+
+  .work-list {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 20px;
+    height: 30px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 300;
+    font-size: 15px;
+  }
+
+  .see11 {
+    display: flex;
+    justify-content: center;
+    height: 51px;
+    width: 215px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    margin-bottom: 0;
+  }
+
+  /* info work4 */
+  .project-name4 {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    width: 100%;
+    margin: 0 auto 20px 0;
+    padding: 0;
+    text-overflow: ellipsis;
+  }
+
+  .box4 {
+    display: flex;
+    gap: 5px;
+    margin-left: 0;
+    margin-bottom: 15px;
+    line-height: 100%;
+    font-size: 10px;
+    padding-right: 0;
+    text-overflow: ellipsis;
+    margin-right: 0;
+  }
+
+  .see4 {
+    display: flex;
+    justify-content: center;
+    height: 51px;
+    width: 212px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    margin-left: 0;
+    margin-bottom: 0;
+  }
+
+  /* About section */
+  .about {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    justify-content: space-between;
+    align-items: center;
+    width: 80%;
+    height: 100%;
+    margin: 0;
+    gap: 0;
+    padding: 10% 0;
+    max-width: 1120px;
+  }
+
+  .colorboxs {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: 0;
+    width: 50%;
+    min-width: 597px;
+    max-width: 600px;
+    background-image: url(./images/image_dots_with_teal.png);
+    background-position-x: right;
+    background-position-y: bottom;
+    background-size: auto;
+    background-repeat: no-repeat;
+  }
```

Comment:
- [ ] (Optional) You have implemented all the changes suggested so there is no reason not to approve your project. I can tell that you are open to feedback and that is a wonderful trait. I would like to suggest that you make the desktop version responsive so that there would be no horizontal scrolling, especially for this section. You know it is good accessibility practice to prevent horizontal scrolling unless it is necessary. Kindly work on it so that your portfolio is presentable to recruiters when you start searching for a job. 👍